languages/php/example.php

```diff
@@ -3,45 +3,101 @@
 require_once 'vendor/autoload.php';
 
 $access_token = getenv('ACCESS_TOKEN');
+$state_file = getenv('STATE_FILE');
 $organization_id = getenv('ORGANIZATION_ID');
 
 // Configuring the URLS is optional, set them to null to use the default values
 $api_url = getenv('API_URL');
 $identity_url = getenv('IDENTITY_URL');
 
-$client_settings = new \Bitwarden\Sdk\BitwardenSettings($api_url, $identity_url);
+try {
+    $client_settings = new \Bitwarden\Sdk\BitwardenSettings($api_url, $identity_url);
 
-$bitwarden_client = new \Bitwarden\Sdk\BitwardenClient($client_settings);
-$bitwarden_client->access_token_login($access_token);
+    $bitwarden_client = new \Bitwarden\Sdk\BitwardenClient($client_settings);
 
-// create project
-$res = $bitwarden_client->projects->create('php project', $organization_id);
-$project_id = $res->id;
+    $bitwarden_client->auth->login_access_token($access_token, $state_file);
 
-// get project
-$res = $bitwarden_client->projects->get($project_id);
+    // create project
+    print("Projects:\n");
+    $res = $bitwarden_client->projects->create($organization_id, 'php project');
+    $project_id = $res->id;
+    print("\tcreate: '" . $project_id . "'\n\n");
 
-// list projects
-$res = $bitwarden_client->projects->list($organization_id);
+    // get project
+    $res = $bitwarden_client->projects->get($project_id);
+    print("\tget: '" . $res->name . "'\n\n");
 
-// update project
-$res = $bitwarden_client->projects->put($project_id, 'php test awesome', $organization_id);
+    // list projects
+    $res = $bitwarden_client->projects->list($organization_id);
+    print("\tlist:\n");
+    foreach ($res->data as $project) {
+        print("\t\tID: '" . $project->id . "', Name: '" . $project->name . "'\n");
+    }
+    print("\n");
 
-// create secret
-$res = $bitwarden_client->secrets->create("New Key", "hello world", $organization_id, [$project_id], "123");
-$secret_id = $res->id;
+    // update project
+    $res = $bitwarden_client->projects->update($organization_id, $project_id, 'php test awesome');
```

Comment:
Do we want this as the name? I don't mind but maybe we want something more similar to the other bindings?